<launch>
    <arg name = "color_topic" value = "/d400/color/image_raw"/>
    <arg name = "depth_topic" value = "/d400/aligned_depth_to_color/image_raw"/>
    <node name="aruco_localizator" pkg="aruco_localization_v2" type="aruco_localization_v2" output="screen">
        <param name = "color_topic" type = "str" value = "$(arg color_topic)"/>
        <param name = "depth_topic" type = "str" value = "$(arg depth_topic)"/>
    </node>
</launch>